app/controllers/subjects_controller.rb

```diff
@@ -15,7 +15,9 @@ def show
   def all
     subjects =
       if params[:search].present?
-        Subject.where("lower(unaccent(name)) LIKE lower(unaccent(?))", "%#{params[:search].strip}%")
+        Subject
```

Comment:
Just as a comment, I think this could also be written like this

```ruby
Subject
  .where(
    "lower(unaccent(name)) LIKE lower(unaccent(:search)) OR lower(unaccent(short_name)) LIKE lower(unaccent(:search))", 
    search: "%#{params[:search].strip}%"
  )
```

which I just like a bit more as I feel it more readable.

Reply:
Will go with this approach and investigate a different approach once I start working on searching subjects by code 🙂